build.gradle

```diff
@@ -32,9 +32,9 @@ task coverage(type: JacocoReport) {
     classDirectories.from files(sourceSets.main.output)
     executionData.from files(jacocoTestReport.executionData)
     afterEvaluate {
-        classDirectories.from files(classDirectories.files.collect {
-            fileTree(dir: it, exclude: ['**/*.jar'])
-        })
+        classDirectories.setFrom(files(classDirectories.files.collect {
+            fileTree(dir: it, exclude: ['ay2021s1_cs2103_w16_3/finesse/ui/**/*', '**/*.jar'])
```

Comment:
- `setFrom` is used to override the previous value so that the excludes take effect. Otherwise, `from` simply appends the files.
- I am not sure why `**/*.jar` has been excluded previously, but I don't want to touch it so it has been left that way.